app/assets/stylesheets/govuk_publishing_components/components/_phase-banner.scss

```diff
@@ -6,3 +6,18 @@
     margin-right: govuk-spacing(2);
   }
 }
+
+.gem-c-phase-banner--inverse {
+  background: $govuk-brand-colour;
+
+  .govuk-phase-banner__content__tag {
+    background: govuk-colour("white");
+    color: $govuk-brand-colour;
+  }
+
+  .govuk-phase-banner__content__app-name,
+  .govuk-phase-banner__text,
+  .govuk-phase-banner__text a {
+    color: govuk-colour("white");
```

Comment:
Unrelated to this PR, I've just noticed the focus state on these links isn't our standard yellow outline, looks like the browser default.

Reply:
This is because the links don't use `govuk-link` - I've changed this on the defaults and in the examples.